src/deepsparse/v2/routers/router.py

```diff
@@ -105,3 +124,37 @@ def validate(operators: List[Operator]) -> bool:
                 """
                 return False
         return True
+
+
+class TextGenerationRouter(Router):
```

Comment:
what's the reasoning again we didn't name this `GraphRouter`?

Reply:
+1

Reply:
Nothing in particular. Thought this router may end up being text-gen specific but in the end it came out to be fairly generic. Will change

Reply:
done